R/call_derivation.R

```diff
@@ -7,7 +7,7 @@
 #' @param derivation The derivation function to call
 #' @param variable_params A `list` of arguments that are different across iterations.
```

Comment:
I would add A `list` of **function** arguments that are different across iterations.